### ISSUE TITLE
Comment on a git commit

### ISSUE THREAD

**Issue by jasikpark** [mshick/add-pr-comment#68]
I'm building a deploy preview commenting step and initially I didn't add `if: ${{ github.event.pull_request }}` to my comment steps so my comments were getting updated with the main branch deploy links. Adding the if statement works for that, but I realize I'd like to comment on the commit to `main` w/ the deploy status.

I realize that's not something this action explicitly supports, but it'd be useful. I wonder if I should explore how to do action status links like Netlify and other apps do instead??

**Comment by mshick** [comment]
@jasikpark I agree, that would be very helpful. The way I've handled that situation in the past is to use a workflow that updates the `main` branch `README.md` with the message I want. 

If you could explore how to do what you're saying and share some links to relevant GitHub APIs I could definitely look at adding a feature for this.
